src/main/java/seedu/ccacommander/model/ModelManager.java

```diff
@@ -197,8 +197,8 @@ public ObservableList<Event> getFilteredEventList() {
     }
 
     @Override
-    public ObservableList<Attendance> getFilteredAttendanceList() {
-        return filteredAttendances;
+    public ObservableList<Enrolment> getFilteredEnrolmentList() {
+        return filteredEnrolments;
```

Comment:
Should include a test for this as part of the PR.

Reply:
Added